docs/tool-configuration/configuration-helm.md

```diff
@@ -29,145 +29,44 @@ helm:
 
 ## CLI Configuration
 
--------------------
-### namespace
-* **BitOps Property:** `namespace`
-* **Environment Variable:** `NAMESPACE`
-* **default:** `""`
-* **required:** yes
-
-namespace scope for this request
-
--------------------
-### timeout
-* **BitOps Property:** `timeout`
-* **Environment Variable:** `TIMEOUT`
-* **default:** `"500s"`
-
-time to wait for any individual Kubernetes operation (like Jobs for hooks) 
-
--------------------
-### set
-* **BitOps Property:** `set`
-* **Environment Variable:** `HELM_SET_FLAG`
-* **default:** `{}`
-
-list of "key=value" strings to pass in to `helm` via `--set`
-
--------------------
-### debug
-* **BitOps Property:** `debug`
-* **Environment Variable:** `HELM_DEBUG`
-* **default:** `""`
-
-enable verbose helm output
-
--------------------
-### atomic
-* **BitOps Property:** `atomic`
-* **Environment Variable:** `TODO`
-* **default:** `""`
-
-if set, the installation process deletes the installation on failure
-
--------------------
-### force
-* **BitOps Property:** `force`
-* **Environment Variable:** `TODO`
-* **default:** `""`
-
-sets helm's `--force` flag
-
--------------------
-### dry-run
-* **BitOps Property:** `dry-run`
-* **Environment Variable:** `TODO`
-* **default:** `""`
-
-simulate an install
+| Property  | Environment Variable | Description                                                  | Default | Required |
+| --------- | -------------------- | ------------------------------------------------------------ | ------- | -------- |
+| namespace | NAMESPACE            | Namespace scope for this project                             | `null`  | Yes      |
+| timeout   | TIMEOUT              | Time to wait for any individual Kubernetes operation (like Jobs for hooks) | `500s`  |          |
+| set       | HELM_SET_FLAG        | List of "key=value" strings to pass in to `helm` via `--set` | `{}`    |          |
+| debug     | HELM_DEBUG           | Enable verbose helm output                                   | `null`  |          |
+| atomic    |                      | If set, the installation process deletes the installation on failure | `null`  |          |
+| force     |                      | Sets helm's `--force` flag                                   | `null`  |          |
+| dry-run   |                      | Simulate an install                                          | `null`  |          |
```

Comment:
Looking at the tables, does it make sense to move the `Default` column right before `Description`?
I mean, in the context of Key=Value this is easier to understand.

![image](https://user-images.githubusercontent.com/1533818/199080444-ad4d1f7a-d5da-46d2-8091-46d3734bf2d1.png)

@ccapell any reasons behind composing it that way?
Curious what other folks think @dylan-bitovi @LeoDiazL @mickmcgrath13 @PhillypHenning

Reply:
Love the idea of the table. Gives you a cleaner view.

Regarding default position, it does makes sense next to the environment variable, as long as it adds something. If it's going to be mainly null or a default you won't change, I'll leave it there.

Reply:
@armab I was just adding the parameters in the order that they appeared.  Most of the tables have very few if any default values.  For consistency stake, all the tables have the same order.